### PR TITLE
Minor fixes to CONTRIBUTING.md

### LINKED ISSUE #2578: Minor fixes to CONTRIBUTING.md
**Current Behavior**
- The links to `# Contributing to Layer5 Projects` and `# Contributing to Layer5's Blogs` in the table of contents doesn't work due to changes in [earlier commit](https://github.com/layer5io/layer5/commit/1059861ebae3cc642489a8c11bb2e54175fb1c0d#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R37)
- Section for  ` [Common Types of Site Updates](#common-types-of-site-updates)` [this ](https://github.com/layer5io/layer5/blame/master/CONTRIBUTING.md#L12) doesn't exist  in the table of contents
- Alias `commit = commit -s` doesn't work (probably because git commit is an existing command)

**Desired Situation**
- Fix the table of content links 
- Remove `Common Types of Site Updates` 
- Update the alias to `cm = commit -s`  and remove  `cm = commit -s -m`

### PR DESCRIPTION
**Pull request by lem0n4id**
**Description**
Changes done

- [x] Fix broken links in the table of contents
- [x] Remove `Common Types of Site Updates` from the table of contents
- [x] Update the alias from `cm = commit -s -m` to `cm = commit -s`
- [x] updated how to clone the fork of the repo

This PR fixes #2578 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -152,7 +151,7 @@ Follow the following instructions to start contributing.
   **2.** Clone your forked copy of the project.
 
   ```
-  git clone https://github.com/layer5io/layer5.git
+  git clone https://github.com/<your-github-username>/layer5.git
```
Comment (leecalcote) [inline CONTRIBUTING.md:154]: This change is incorrect.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -275,8 +274,7 @@ To ensure all your commits are signed, you may choose to add this alias to your
   ```
   [alias]
     amend = commit -s --amend
-    cm = commit -s -m
-    commit = commit -s
+    cm = commit -s
```
Comment (leecalcote) [inline CONTRIBUTING.md:277]: Why the change here?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -3,13 +3,12 @@
 We are beyond excited to see that you want to contribute! We would love to accept your contributions. Layer5 is built by the community and warmly welcomes collaboration. There are many ways in which one could contribute to Layer5 and every contribution is equally appreciated here. Navigate through the following to understand more about contributing here. 
 
   - [Before You Get Started](#before-you-get-started)
-  - [Contributing to Layer5's Code](#contributing-to-layer5s-code)
-  - [Contributing to Layer5's Docs](#contributing-to-layer5s-docs)
+  - [Contributing to Layer5 Projects](#contributing-to-layer5-projects)
+  - [Contributing to Layer5's Docs](#contributing-to-layer5s-blogs)
```
Comment (leecalcote) [inline CONTRIBUTING.md:7]: The hyperlink works as-is, correct? This change seems to break the link. 

The title of the link can be updated from "Docs" to "Blogs".

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (lem0n4id) [reply CONTRIBUTING.md:277]: I added this as an alternative to the original `commit = commit -s` which doesnt work

File: CONTRIBUTING.md (hunk 1 above)
Comment (lem0n4id) [reply CONTRIBUTING.md:154]: doing `git clone https://github.com/layer5io/layer5.git` would just clone the original repo and **not the fork**

File: CONTRIBUTING.md (hunk 2 above)
Comment (warunicorn19) [reply CONTRIBUTING.md:277]: this is not a required change, I have this alias configured and it works.

Comment (lem0n4id) [reply CONTRIBUTING.md]: Weird. Okay, I had reverted the change here

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -275,7 +274,7 @@ To ensure all your commits are signed, you may choose to add this alias to your
   ```
   [alias]
     amend = commit -s --amend
-    cm = commit -s -m
+    cm = commit -m
```
Comment (warunicorn19) [inline CONTRIBUTING.md:277]:
```suggestion
    cm = commit -s -m
```

**Comment by warunicorn19** [comment]
Thank you @lem0n4id
